src/backend/vector/avx2/field.rs

```diff
@@ -539,13 +536,18 @@ impl FieldElement2625x4 {
         };
 
         // Perform two halves of the carry chain in parallel.
-        carry(&mut z, 0); carry(&mut z, 4);
-        carry(&mut z, 1); carry(&mut z, 5);
-        carry(&mut z, 2); carry(&mut z, 6);
-        carry(&mut z, 3); carry(&mut z, 7);
+        carry(&mut z, 0);
+        carry(&mut z, 4);
+        carry(&mut z, 1);
+        carry(&mut z, 5);
+        carry(&mut z, 2);
+        carry(&mut z, 6);
+        carry(&mut z, 3);
+        carry(&mut z, 7);
         // Since z[3] < 2^64, c < 2^(64-25) = 2^39,
         // so    z[4] < 2^26 + 2^39 < 2^39.0002
-        carry(&mut z, 4); carry(&mut z, 8);
+        carry(&mut z, 4);
+        carry(&mut z, 8);
```

Comment:
The original formatting illustrates how two halves of the carry chain are being computed in parallel, and it's very clear which half is which prior to rustfmt, which loses that information.